Ports/libunistring/package.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=libunistring
+version=1.0
+files="https://ftpmirror.gnu.org/gnu/libunistring/libunistring-${version}.tar.gz libunistring-${version}.tar.gz
+https://ftpmirror.gnu.org/gnu/libunistring/libunistring-${version}.tar.gz.sig libunistring-${version}.tar.gz.sig
+https://ftpmirror.gnu.org/gnu/gnu-keyring.gpg gnu-keyring.gpg"
```

Comment:
Same remark; use a SHA-256 hash instead of a signature downloaded from the same source.